husky_components/input/main.js

```diff
@@ -200,6 +200,13 @@ define([], function() {
          * Binds Dom-related events
          */
         bindDomEvents: function() {
+            this.sandbox.dom.on(this.$el, 'change', function(event) {
+                event.stopPropagation();
+                event.preventDefault();
+
+                this.$el.trigger('change');
```

Comment:
Referring to your other comment: What you are basically doing here is to catch the event emitted on `this.$el`, and triggering again, so how can this improve the situation?